src/tests/totp/bin/Makefile

```diff
@@ -3,26 +3,29 @@
 #
 all: totp
 
-include ../../../Make.inc
+include ../../../../Make.inc
 
 #
 #  Hack up stuff so we can build in a subdirectory.
 #
 .PHONY: src
 src:
-	@ln -sf ../../../src
+	@ln -sf ../../../../src
 
 .PHONY: freeradius-devel
 freeradius-devel:
-	@ln -sf ../../../src/include freeradius-devel
+	@ln -sf ../../../../src/include freeradius-devel
```

Comment:
I'm not sure why this is necessary?  the `src` directory is only 3 directories up.  If I do:

```
$ cd src/modules/rlm_totp
$ ls -al ../../../src/
```

I get the contents of the `src` directory, as expected.

Reply:
@alandekok That `Makefile` is in `src/tests/totp/bin/`.

so, we need to go up 4 levels. e.g: 

```
$ (cd src/tests/totp/bin/ && ls -adl ../../../../src/)
drwxr-xr-x  16 jpereira  staff  512 Sep 14 22:28 ../../../../src/
$
```

Reply:
I replaced by `$(top_srcdir)`